sypht/client.py

```diff
@@ -510,15 +510,28 @@ def list_entities(self, entity_type, company_id=None, page=None, limit=None, end
             params["limit"] = int(limit)
         return self._parse_response(self.requests.get(endpoint, headers=headers, params=params))
 
-    def get_all_entity_ids(self, entity_type, company_id=None, endpoint=None):
+    def get_all_entity_ids(self, entity_type, verbose=True, company_id=None, endpoint=None):
+        """Get all entity_ids for specified entity_type.
+
+        Returns list of objects if verbose (by default):
+        [{"entity_id": "id_0"}, {"entity_id": "id_1"}, ...]
+
+        Returns list of entity_id if not verbose:
+        ["id_0", "id_1", ...]
+        """
         entity_ids = []
         next_page = True
         while next_page:
             if next_page is True:
                 next_page = None  # first page request
             res = self.list_entities(entity_type, company_id, page=next_page)
             next_page = res.get("next_page")
-            entity_ids.extend([{"entity_id": entity_id} for entity_id in res.get("entities")])
+            if verbose:
+                entity_ids.extend(
+                    [{"entity_id": entity_id} for entity_id in res.get("entities")]
+                )
+            else:
+                entity_ids.extend(res.get("entities"))
```

Comment:
Not sure how much value the added `entity_id` keys when `verbose==True` provides. Would opt for just the non-verbose. Thoughts

Reply:
It's related to other endpoints such as `get_many_entities` and `put_many_entities` which required verbose format.

Reply:
Yep, makes sense. Happy to leave it in